pulsar-broker/src/test/java/org/apache/pulsar/broker/service/BatchMessageTest.java

```diff
@@ -217,7 +214,8 @@ public void testSimpleBatchProducerWithFixedBatchTime(CompressionType compressio
         consumer.close();
 
         Producer<byte[]> producer = pulsarClient.newProducer().topic(topicName).compressionType(compressionType)
-                .batchingMaxPublishDelay(10, TimeUnit.MILLISECONDS).enableBatching(true)
+                .batchingMaxPublishDelay(Integer.MAX_VALUE, TimeUnit.MILLISECONDS)
```

Comment:
We should not change here. This will cause the test to hang forever.

Reply:
Good catch!